ui/imports/utils/Backpressure/Backpressure.qml

```diff
@@ -21,13 +21,13 @@ Item {
             owner = backpressure;
         }
 
-        owner.Component.destruction.connect(cleanup);
+        owner.Component.onDestruction.connect(cleanup);
```

Comment:
Are you sure? The correct syntax should be `object.signal.connect(slot)`. See https://doc.qt.io/qt-6/qtqml-syntax-signals.html#connecting-signals-to-methods-and-signals

Reply:
Surprisingly, both versions work.

```
Item {
    id: foo

    Component.onCompleted: {
        foo.Component.onDestruction.connect(() => console.log("onDestruction"))
        foo.Component.destruction.connect(() => console.log("destruction"))
    }
}
```
The output is:
```
qml: onDestruction
qml: destruction
```

I have updated the function to `onDestruction` in order to maintain consistency with the [original library](https://github.com/e-fever/backpressure/blob/2cb0e8b9b7a59f28a238f138b7bff96e834f1d1e/Backpressure/Backpressure.qml#L24). Consequently, I mistakenly assumed that our current implementation is incorrect.

Since both versions work, I will keep it as is.

Reply:
When we chat yesterday, I've thought that you were thinking about to remove that line (cause I understood you it's not needed), not to change `destruction` to `onDestruction`.

> Surprisingly, both versions work.

didn't know that